core/src/main/java/io/questdb/griffin/engine/functions/bool/InSymbolCursorFunctionFactory.java

```diff
@@ -172,7 +185,7 @@ public void toPlan(PlanSink sink) {
         private void buildValueSet() {
             final Record record = cursor.getRecord();
             while (cursor.hasNext()) {
-                CharSequence value = func.get(record, 0);
+                CharSequence value = func.get(record, 0, sink);
```

Comment:
You can use `Misc.getThreadLocalSink()` here instead of the dedicated sink. Thread static getter will be used once, relatively not expensive.

Reply:
I was considering this. but then I decided it was asking for troubles: we don't know what `func.get()` will do. if it's an actual physical record then it's safe. but it can be a virtual record backed by an arbitrary function. and if that function uses a thread-local sink too then things will break in a spectacular (and hard to analyze) way. 

wdyt?